sample/source/scenes/GraphicsTest.cpp

```diff
@@ -46,14 +46,46 @@ void GraphicsTest::Load(ecs::Ecs world, ModuleProvider modules)
         "skeleton\\BaseColor.nca",
         "skeleton\\Roughness.nca",
         "skeleton\\Metallic.nca",
+        "denim\\BaseColor.nca",
```

Comment:
For linux to be able to run the sample, we'll have to start using '/' everywhere. I've fixed the existing ones in an in-progress branch. Depending on merge order, one of us will have to change these new ones.

Reply:
I'll do it now